public/_posts/_articles/marco-worms/yearn-governance/es.md

```diff
@@ -0,0 +1,128 @@
+
+Según lo establecido por [YIP-55](https://gov.yearn.finance/t/yip-55-formalize-the-yip-process/7959), aquí hay una forma estándar de escribir propuestas:
+
+**Resumen**:
+El resultado del cambio propuesto, escrito de manera no técnica y accesible.
+
+- **Estado:** *(Este campo no es obligatorio pero ayuda a todos)*
+    - **Discusión:** Esta propuesta se encuentra actualmente en fase de discusión. De acuerdo con nuestras reglas de votación descritas en YIP-55, estará en discusión durante al menos 3 días con una encuesta de foro no vinculante para medir el sentimiento antes de que se le pueda asignar un número de YIP y pasar a snapshot para una votación vinculante.
+    - **Votación:** Esta propuesta se encuentra actualmente en la fase de votación. Emite tu voto en snapshot. Puede conocer nuestras reglas de votación en YIP-55
+    - **Aprobada:** Esta propuesta fue aprobada el 25 de abril de 2021 a las 7:00 UTC con un 99,97 % de votos a favor.
+
+**Resumen:**
+Una breve descripción (~200 palabras) del cambio propuesto. Debe describir claramente lo que sucede si se aprueba la propuesta.
+
+**Motivación:**
+El problema actual que te hizo escribir la propuesta de cambio.
+
+**Especificación:**
+Una descripción larga de cómo esta propuesta resuelve el problema, debe describir cómo implementar el YIP
+
+**Encuesta:**
+Una dinámica de votación que se crea en el foro de gobierno para medir el sentimiento de la comunidad sobre la propuesta debe ser superior al 50 %.
+
+### Discusión
+
+Debe haber suficiente tiempo (al menos 3 días) de discusión para que los miembros de la comunidad puedan abordar la propuesta desde muchos ángulos. Utiliza la herramienta de encuesta integrada de Discourse para evaluar el sentimiento de la comunidad sobre la propuesta antes de realizar una votación formal; la encuesta debe tener al menos un 50 % de votos a favor.
+
+![](./image3.jpg?w=900&h=351)
+
+Lo ideal sería que las discusiones tomaran el tiempo necesario para aclarar lo que se propuso a la comunidad. Hablar con diferentes partes interesadas **antes** de proponer algo es una buena manera de medir el sentimiento y lo ayuda a presentar ideas complejas con un mejor apoyo desde el principio. Un ejemplo de propuesta con excelente discusión es [YIP-57](https://gov.yearn.finance/t/yip-57-funding-yearns-future/9319) que acuñó 6,666 tokens YFI más allá del suministro inicial de 30,000 y tuvo para alinear diferentes tipos de partes interesadas, desde miembros de DAO hasta grandes titulares de YFI.
+
+### Votación
+
+Después de pasar el paso de discusión, la propuesta se convierte en un YIP (o tal vez un YDP dependiendo de su contenido) y se someterá a la fase de votación. Para enviar un voto en [Página de snapshots de Yearn](https://yearn.snapshot.page/#/) debe tener al menos 1 $ YFI o pedirle a alguien que lo haga por ti, si la propuesta es sólida, esto no será una barrera. ¡Comuníquete con un moderador del foro para obtener un número oficial para el nuevo YIP!
```

Comment:
AS-IS: 1 $ YFI
TO-BE: 1 $YFI

to avoid confusion with 1 dollar and 1 yfi.



As-IS: ¡Comuníquete con un moderador del foro para obtener un número oficial para el nuevo YIP!

TO-BE: ¡Comunícate con un moderador del foro para obtener un número oficial para el nuevo YIP!

Reply:
Done